src/overlays/actors/ovl_En_Tubo_Trap/z_en_tubo_trap.h

```diff
@@ -5,11 +5,20 @@
 
 struct EnTuboTrap;
 
+typedef void (*EnTuboTrapActionFunc)(struct EnTuboTrap*, GlobalContext*);
+
 typedef struct EnTuboTrap {
-    /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x60];
+    /* 0x0000 */ Actor actor;
+    /* 0x0144 */ EnTuboTrapActionFunc actionFunc;
+    /* 0x0148 */ f32 targetY;
+    /* 0x014C */ Vec3f originPos;
+    /* 0x0158 */ ColCylinder collider;
 } EnTuboTrap; // size = 0x1A4
 
 extern const ActorInit En_Tubo_Trap_InitVars;
 
+extern u32 D_05018090; // pot break shard displaylist
+
+extern u32 D_05017EA0; // the pot displaylist
```

Comment:
These should be of type `Gfx[]`

Also, since these are probably from `gameplay_dangeon_keep`, put them in `include\segment.h` instead. That file is a little messy, but the idea is that since multiple actors could be using those symbols, they should defined in one place. Eventually there will be a header file for each object.

Reply:
`extern Gfx[] D_05018090` wont build if placed in `include/segment.h` but `extern UNK_TYPE D_05018090` does.

is it maybe too early in the build process and Gfx isn't defined yet?

Reply:
Since it's an array type, the `[]` goes after the name, like `extern Gfx D_05018090[]`